gensrc/thrift/Data.thrift

```diff
@@ -118,6 +118,7 @@ struct TStatisticData {
     14: optional i64 partitionId
     // the batch load version
     15: optional binary hll
+    16: optional string partitionName
```

Comment:
do not need dbName, catalog name?

Reply:
this is used to return statistics result to fe, fe knows dbName and catalog name when it query the statistics, so there is no need to return dbname and catalog name.